Cargo.toml

```diff
@@ -31,7 +31,7 @@ members = [
   "samples/mixed",
   "samples/property",
   "samples/seat_massager",
-  "samples/streaming",
+  # "samples/streaming",
```

Comment:
This was mentioned in the PR's notes:

The streaming sample uses an OSS license that we so not wish to use. Its build step has been deactivated until it can be changed to no longer depend on the show-image crate.